src/themes/light-theme.js

```diff
@@ -59,6 +59,38 @@ const mono1000 = '#000000';
 const primaryFontFamily = '"Helvetica Neue", arial, sans-serif';
 
 export default {
+  border: {
```

Comment:
We already have a `borders` key in theme, should we consolidate? May also need to update `ThemeT` in src/styles/types.js

Reply:
Ah, that may have been added between my initial add and now. It looks like they can be pretty cleanly merged- the existing `borders` key is strictly concerned with border radius. I'll merge.